CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Add certificate issuer label to `cert_exporter_secret_not_after` and `cert_exporter_not_after` metrics.
```

Comment:
Just for my understanding:
Why do we need the issuer label if we have the certificate CR name?
Does it make sense to only expose the certificate CR name if it exists?

Reply:
The issuer of a certificate usually is not equal to the Certificate CR name and looks usally like this: `OU=corp, O=mycorp, L=New York, C=US`

Here `label` is a label of the metric not a kubernetes label, so maybe the Changelog entry is worded incorrect?

Reply:
I think the changelog entry is correct. I was mainly thinking from an operational side:
When I look at an alert, do I really need both? But I assume it's mainly so we have the certificate CR name and the corp which created it and can then exclude those which are not from giant swarm?

Reply:
Yeah the issuer is there to help the oncaller to see if issuer is lets encrypt (probably GS managed) or not (probably not).

The main improvement I want to use is the availability of the certificate cr name. (Change it to not alert if the certificate cr name is empty)